ui/appui-react/src/appui-react/frontstage/FrontstageConfig.ts

```diff
@@ -44,6 +44,8 @@ export interface FrontstageConfig extends CommonProps {
   readonly contentManipulation?: WidgetConfig;
   /** The settings of the active tool. */
   readonly toolSettings?: WidgetConfig;
```

Comment:
I'd say let's fold `emptyMessage` to `toolSettings`. You can simply use a type union: `WidgetConfig & { x?: number }` or introduce a new type.

Reply:
- Changed to use type union.
- Changed from `string` to `ReactNode`.